guides/users/user-metadata.mdx

```diff
@@ -0,0 +1,103 @@
+---
+title: User Metadata
+---
+
+Hanko allows for defining arbitrary user metadata. Metadata can be categorized into
+three types that differ as to how they can be accessed and modified:
+
+| Metadata type | Public API                   | Admin API             |
+|---------------|------------------------------|-----------------------|
+| Private       | No read or write access      | Read and write access |
+| Public        | Read access                  | Read and write access |
+| Unsafe        | Read access and write access | Read and write access |
+
+Each metadata type is currently limited to 3000 characters.
+
+## Private metadata
+
+Private metadata should be used for sensitive data that should not be exposed to the client (e.g., internal flags/ids,
+configuration, or access control details).
+
+### Accessing private Metadata
+
+Private metadata can be read through the Admin API only using the
+[Get metadata of a user](/api-reference/admin/user-management/get-metadata-of-a-user)
+endpoint.
+
+### Setting private metadata
+
+Private metadata can be set and modified through the Admin API only by using the
+[Patch metadata of a user](/api-reference/admin/user-management/patch-metadata-of-a-user) endpoint.
```

Comment:
When I click on the link, I get to the correct page, but the navigation menu, on the left, does not indicate where I am.